### PR TITLE
New definition for animejs

### PR DESCRIPTION
**Pull request by A-Babin**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not provide its own types, and you can not add them.
- [X] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/animejs/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (andy-ms) [inline types/animejs/tsconfig.json]: Base this on `abs/tsconfig.json`. Just change "files".

File: types/animejs/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+// Project: http://animejs.com
+// Definitions by: Andrew Babin <https://github.com/A-Babin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type FunctionBasedParamter = (element : HTMLElement, index : number, length : number) => number;
+type AnimeCallbackFunction = (anim : anime.AnimeInstance) => void;
+type AnimeTarget = string | object | HTMLElement | SVGElement | NodeList | null; //Allowing null is necessary because DOM querys may not return anything. 
+
+declare namespace anime {
+	const enum EasingsEnum {
```
Comment (andy-ms) [inline types/animejs/index.d.ts]: String enums are a ts2.4 feature, so you'll need to add a `// TypeScript Version: 2.4` comment on line 5 (just under the header).

Comment (mhegazy) [inline types/animejs/index.d.ts]: do not use string enums in declaration files. they will break under `--isolatedModule`.

**Review by mhegazy — CHANGES_REQUESTED** [review]

**Review by mhegazy — APPROVED** [review]

**Comment by A-Babin** [comment]
Great! Thanks @andy-ms and @mhegazy for the assistance.

**Pull request merged**